### PR TITLE
Fixup indentations changes

### PR DESCRIPTION
**Pull request by MihaZupan**
#1382 and #1352 introduced undesired/missing indentations.

I wrote a program to apply file scoped namespace changes from before #1352 and #1382, and then tried cleaning up any small style changes included in those PRs that we do want to keep. There may be a few lines here that we could change, but I did as best I could re-reading the 5k diff a bunch of times.

### CODE REVIEW

**Comment by Tratcher** [comment]
> I wrote a program to apply file scoped namespace changes

😁 Have you tried `dotnet format`?

File: test/ReverseProxy.Tests/SessionAffinity/BaseSesstionAffinityPolicyTests.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ public class BaseSesstionAffinityPolicyTests
         var cluster = new ClusterState("cluster");
         var affinityResult = provider.FindAffinitizedDestinations(context, cluster, _defaultOptions, allDestinations);
 
-        if (unprotectCalled)
+        if(unprotectCalled)
```
Comment (Tratcher) [inline test/ReverseProxy.Tests/SessionAffinity/BaseSesstionAffinityPolicyTests.cs:49]: Revert

**Review by Tratcher — APPROVED** [review]

File: src/ReverseProxy/WebSocketsTelemetry/WebSocketsTelemetryStream.cs [hunk 2]
```diff
@@ -52,12 +52,12 @@ public WebSocketCloseReason GetCloseReason(HttpContext context)
...
-            ForwarderError.UpgradeResponseClient => WebSocketCloseReason.ClientDisconnect,
-            ForwarderError.UpgradeResponseCanceled => WebSocketCloseReason.ClientDisconnect,
-            ForwarderError.UpgradeRequestDestination => WebSocketCloseReason.ServerDisconnect,
-            ForwarderError.UpgradeResponseDestination => WebSocketCloseReason.ServerDisconnect,
+            ForwarderError.UpgradeRequestClient         => WebSocketCloseReason.ClientDisconnect,
+            ForwarderError.UpgradeRequestCanceled       => WebSocketCloseReason.ClientDisconnect,
+            ForwarderError.UpgradeResponseClient        => WebSocketCloseReason.ClientDisconnect,
+            ForwarderError.UpgradeResponseCanceled      => WebSocketCloseReason.ClientDisconnect,
+            ForwarderError.UpgradeRequestDestination    => WebSocketCloseReason.ServerDisconnect,
+            ForwarderError.UpgradeResponseDestination   => WebSocketCloseReason.ServerDisconnect,
```
[the remark this answers is not in the document]
Comment (MihaZupan) [reply src/ReverseProxy/WebSocketsTelemetry/WebSocketsTelemetryStream.cs:60]: Really?

I find code like this impossible to read in comparison:
image

image

What maintenance effort do you mean?

**Comment by MihaZupan** [comment]
> Have you tried dotnet format?

Didn't think to before, but it looks like that brings along a lot of unrelated changes and I can't get it to do the scoped filename transformation

File: src/ReverseProxy/WebSocketsTelemetry/WebSocketsTelemetryStream.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (Tratcher) [reply src/ReverseProxy/WebSocketsTelemetry/WebSocketsTelemetryStream.cs:60]: It's a style that requires a lot of manual developer setup, and when you add a longer entry later you have to reformat everything else. Formatting tools are also likely to break it. The readability is nice, but not worth the effort.

I wouldn't block any individual PR on a style issue like this, but I do think we should use a consistent approach for this.

**Pull request merged**